tests/core/evaluation/test_marker_tracker_loader.py

```diff
@@ -0,0 +1,59 @@
+from rasa.core.evaluation.marker_tracker_loader import MarkerTrackerLoader
+from rasa.core.tracker_store import TrackerStore
+import pytest
+
+
+# Assuming a TrackerStore with 5 trackers in it
```

Comment:
Not sure the best way of setting this up - is it going to be easiest to create a fixture from scratch with a minimal domain or does something already exist?

Reply:
I saw that you asked this question on slack. Makes sense to find existing trackers and use them to create fixtures for your tests.